platform/ios/Integration Tests/Snapshotter Tests/MGLMapSnapshotterTest.m

```diff
@@ -462,4 +478,60 @@ - (void)testSnapshotWithOverlayHandlerSuccess🔒 {
     [self waitForExpectations:@[expectation] timeout:10.0];
 }
 
+- (void)testSnapshotCoordinatesWithOverlayHandler🔒 {
```

Comment:
Not specific to this PR, but I would love to see a reference as to that 🔒means here.

Reply:
👍